assets/RNAseq_report.Rmd

```diff
@@ -1425,7 +1445,9 @@ for (file in contrast_files){
     fname <- tools::file_path_sans_ext(basename(file))
 
     dir.create(paste("pathway_analysis", fname, sep="/"))
-    dir.create(paste("pathway_analysis", fname, "pathway_heatmaps", sep="/"))
+    if (!isNot(params$citest)) {
```

Comment:
```suggestion
    if (params$citest) {
```

Reply:
It's easier to understand a positive than a double negative

Reply:
So the problem is that for some reason, the whole combination of R and Rmd with the params "" seems to be working rather imperfectly. When a param is not given but tested for, e.g. params$relevel, I get  `Error in if (params$path_relevel) { : argument is of length zero`. That is why I added the isNot function, it tests for multiple things. The only thing I could to is to write a positive `is` equivalent

Reply:
I agree, a test for true with `is` would be more readable than a double negative